react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ * TODO: The buttons don't work correctly!
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+
+  function toggleBackgroundPrev() {
```

Comment:
This name looks leftover from the Button toggle handler. For its use now, it is a strange name since it doesn't toggle anything and doesn't have anything to do with background. :-)